### PR TITLE
chore: bump kcl to 0.4.2

### PR DESCRIPTION
**Pull request by hectorj2f**
We are bumping the UI and KCL versions

### CODE REVIEW

File: stable/kommander/Chart.yaml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 apiVersion: v1
-appVersion: 1.0.0
+appVersion: 2.21.0
```
Comment (juliangieseke) [inline stable/kommander/Chart.yaml]: Dont change this, it stays 1.0.0.

File: stable/kommander/requirements.yaml [hunk 2]
```diff
@@ -4,11 +4,11 @@ dependencies:
     repository: "https://mesosphere.github.io/charts/staging"
     condition: kubeaddons-catalog.enabled
   - name: kommander-cluster-lifecycle
-    version: "0.4.0"
+    version: "0.4.2"
     repository: "https://mesosphere.github.io/kommander-cluster-lifecycle/charts"
     condition: kommander-cluster-lifecycle.enabled, global.kommander-cluster-lifecycle.enabled
   - name: kommander-ui
-    version: "1.192.3"
+    version: "2.21.0"
```
Comment (juliangieseke) [inline stable/kommander/requirements.yaml]: Correct 👍🏻

**Comment by hectorj2f** [comment]
@jimmidyson that is what I ran
```
kommander git:(hectorj2f/kcl) ✗ helm dep update
Hang tight while we grab the latest from your chart repositories...
...Unable to get an update from the "local" chart repository (http://127.0.0.1:8879/charts):
	Get http://127.0.0.1:8879/charts/index.yaml: dial tcp 127.0.0.1:8879: connect: connection refused
...Successfully got an update from the "kubefed-charts" chart repository
...Successfully got an update from the "kcl" chart repository
...Successfully got an update from the "harbor" chart repository
...Successfully got an update from the "kommander" chart repository
...Successfully got an update from the "mesosphere-staging" chart repository
...Successfully got an update from the "localvolumeprovisioner" chart repository
...Successfully got an update from the "test" chart repository
...Successfully got an update from the "jetstack" chart repository
...Successfully got an update from the "bitnami" chart repository
...Successfully got an update from the "stable" chart repository
Update Complete.
Saving 6 charts
Downloading kubeaddons-catalog from repo https://mesosphere.github.io/charts/staging
Downloading kommander-cluster-lifecycle from repo https://mesosphere.github.io/kommander-cluster-lifecycle/charts
Downloading kommander-ui from repo https://mesosphere.github.io/kommander/charts
Downloading kommander-thanos from repo https://mesosphere.github.io/charts/stable
Downloading kommander-karma from repo https://mesosphere.github.io/charts/stable
Downloading grafana from repo https://kubernetes-charts.storage.googleapis.com
Deleting outdated charts
```

**Comment by hectorj2f** [comment]
@jimmidyson what is the old version https://github.com/mesosphere/charts/pull/419/files#diff-ff0d458a260a52099776c6e29267a688R7 ?

**Comment by jimmidyson** [comment]
@hectorj2f See https://github.com/mesosphere/charts/tree/hectorj2f/kcl/stable/kommander/charts ...

**Comment by jimmidyson** [comment]
Both KCL 0.4.0 and kommander UI 1.192.3 are still left lying around.

**Pull request merged**